### DISCUSSION TITLE
Redirect ryot to OIDC automatically

### DISCUSSION THREAD

**Discussion by kaibagley** [IgnisDa/ryot#941] [Q&A] [answered]
Hey everyone,

I have OIDC set up using Authentik, and have disabled the normal login so that when i visit ryot, I only see the "Login using OpenID Connect" button. However, what would be even nicer is that the page would auto redirect me to Authentik and then back just like Immich or Nextcloud does. 

Is there any way to do this currently?

Thanks!!

**Comment by IgnisDa** [comment]
Ideally you should stay logged in for 90 days, so that you do not need to login at all. Is that not happenning for you?

**Reply by kaibagley** [reply]
Yeah that works fine, but I delete cookies + cache + site data relatively often and have to re-login. Was just thinking it might be nice to go straight to the auth rather than having to click on the only button on the screen

**Reply by IgnisDa** [reply]
Honestly this feels like a user introduced problem. I don't think I want Ryot to have this behaviour.

**Reply by kaibagley** [reply]
Constantly having to re-login isn't a problem, I understand that's a result of Arkenfox/whatever other things I've changed in my browser's settings, I just wanted to know if there was an option similar to "Auto Launch" in Immich or whatever the OAuth redirect option in Nextcloud user_oidc. However I understand if this is undesirable behaviour

**Reply by IgnisDa** [reply, marked as answer]
No, there's no such behaviour.
